src/main/java/me/drex/itsours/claim/list/ClaimList.java

```diff
@@ -88,8 +88,17 @@ private void addRegion(Region region, AbstractClaim claim) {
         return byOwner.get(owner) == null ? new ArrayList<>() : byOwner.get(owner);
     }
 
-    public List<AbstractClaim> get(ServerWorld world, BlockPos pos) {
-        return get(Region.get(pos.getX(), pos.getY())).stream().filter(abstractClaim -> abstractClaim.getWorld().equals(world)).collect(Collectors.toList());
+    public AbstractClaim get(ServerWorld world, BlockPos pos) {
```

Comment:
This has to be annotated with `@Nullable`

Reply:
When I declared it with @Nullable the IDE auto imported `import com.sun.istack.internal.Nullable;`, which spit errors when I tried compiling

Reply:
Why don't you use Jetbrain's Annotations `org.jetbrains.annotations`?